src/windows/win-crowdstrike-fix-bootloop-v2.ps1

```diff
@@ -4,6 +4,51 @@
 $partitionlist = Get-Disk-Partitions
 $actionTaken = $false
 
+function CleanUpRegtransmsAndTxrblfFiles 
+{
+    param(
+        [Parameter(Mandatory = $true)]
+        [string]$GuidSuffix,
+        [Parameter(Mandatory = $true)]
+        [string]$DriveLetter
+    )
+
+    Log-Info "Deleting regtrans-ms and txr.blf files under config\TxR for Windows Server 2016 or newer version..."
+    Log-Info "Checking Windows Build number..."
+    $regKey = "HKLM:\temp_software_hive_$GuidSuffix\Microsoft\Windows NT\CurrentVersion"
+    $currentBuild = (Get-ItemProperty $regKey -Name CurrentBuild).CurrentBuild
+    Log-Info "CurrentBuild: $currentBuild"
+    if ($currentBuild -ge 14393) 
```

Comment:
comment as to why we are checking these and more copious comments in the code

Reply:
Sure, I added a few comments to explain it.